modules/component/src/core/render-event/models.ts

```diff
@@ -1,10 +1,18 @@
 interface BaseRenderEvent {
+  /**
+   * true if the event is emitted by a new source
+   */
   reset: boolean;
+  /**
+   * true if the synchronous event is emitted
+   */
+  sync: boolean;
```

Comment:
Initially I was a bit confused about the term `sync`.
I was thinking that it meant that data (or something else) need to be synchronized.
It could be a malformation on my end, but I would rename this to `synchronous` to avoid confusion.  

Again, it it's probably just me, and if you disagree we can leave it as is.

Reply:
Oh 😅 you're right, "sync" can be an abbreviation for multiple things. I'll rename `sync` to `synchronous` then, to avoid potential confusion.